PROJECT_GOVERNANCE.md

```diff
@@ -0,0 +1,33 @@
+Each project under clj-commons need to have at least a maintainer, and the names of the maintainers should be listed
+ in the `.github/CODEOWNERS` file. The organization as such do not pose any 
+ restrictions on how the individual projects are run, other than what is stated in the 
+ [meta/README](https://github.com/clj-commons/meta/README.md).
+
+  This means that PR review protocol, release cycles, and post-release annoncements are up to the 
+ maintainer of the project, even though we encourage announcing new releases to the main Google Group
+ and the #announcement channel in the Clojurians slack. Also, coding style is up to the maintainers of
+ the project, but clj-commons recommend following [The Clojure Style Guide](https://github.com/bbatsov/clojure-style-guide)
+
+  #Continious Integration
+ As clj-commons as an org is most familiar with [CircleCI](https://circleci.com), we recommend using CircleCI
+ as a projects CI tool.
```

Comment:
What about mentioning that CircleCI is configured at the org level already? Meaning that simply adding the CircleCI config files is sufficient to enable CI.

Reply:
@rborer: good points here and above, PR welcome :)

Reply:
@slipset see #50 :-)